src/site/content/en/blog/add-touch-to-your-site/index.md

```diff
@@ -0,0 +1,810 @@
+states", width="500", height="300" %}
+</figure>
+
+On most mobile browsers *hover* and/or *focus* states will apply to an element
+after it's been tapped.
+
+Consider carefully what styles you set and how they will look to the user after
+they finish their touch.
+
+Note: Anchor tags and buttons may have different behavior
+in different browsers, so assume in some cases **hover**
+will remain and in others **focus** will remain.
+
+### Suppressing default browser styles
+
+Once you add styles for the different states, you'll notice that most browsers
+implement their own styles in response to a user’s touch. This is largely
+because when mobile devices first launched, a number of sites didn’t
+have styling for the `:active` state. As a result, many browsers added
+additional highlight color or style to give the user feedback.
+
+Most browsers use the `outline` CSS property to display a ring around an
+element when an element is focused. You can suppress it with:
+
+```css
+.btn:focus {
+    outline: 0;
+
+    // Add replacement focus styling here (i.e. border)
+}
+```
+
+Safari and Chrome add a tap highlight color which can be prevented with the
+`-webkit-tap-highlight-color` CSS property:
+
+```css
+/* Webkit / Chrome Specific CSS to remove tap
+highlight color */
+.btn {
+  -webkit-tap-highlight-color: transparent;
+}
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/input/touch/states-example.html)
+
+Internet Explorer on Windows Phone has a similar behavior, but is suppressed
+via a meta tag:
+
+```html
+<meta name="msapplication-tap-highlight" content="no">
+```
+
+Firefox has two side effects to handle.
+
+The `-moz-focus-inner` pseudo class, which adds an outline on
+touchable elements, you can remove by setting `border: 0`.
+
+If you are using a `<button>` element on Firefox, you get a gradient
+applied, which you can remove by setting `background-image: none`.
+
+```css
+/* Firefox Specific CSS to remove button
+differences and focus ring */
+.btn {
+  background-image: none;
+}
+
+.btn::-moz-focus-inner {
+  border: 0;
+}
+```
+
+[Try it](https://googlesamples.github.io/web-fundamentals/fundamentals/design-and-ux/input/touch/states-example.html)
+
+{% Aside 'caution' %}
+Only suppress the default styles mentioned above if you have pseudo
+classes for `:hover`, `:active` and `:focus`!
+{% endAside %}
+
+### Disabling user-select
+
+When you're creating your UI there may be scenarios where you want users
+to interact with your elements but you want to suppress the default behavior
+of selecting text on long press or dragging a mouse over your UI.
+
+You can do this with the `user-select` CSS property, but beware that
+doing this on content can be **extremely** infuriating
+for users if they *want* to select the text in the element.
+So make sure you use it with caution and sparingly.
+
+```css
+user-select: none;
+```
```

Comment:
```suggestion
```css
/* Example: dDisable selecting text on a paragraph element: *?
p.disable-text-selection {
  user-select: none;
}
```
```

Reply:
This is a stylistic nit, but it would be invalid CSS to specify just a property without a selector. So I added the selector to make it valid CSS.